core/src/main/java/apoc/util/Utils.java

```diff
@@ -87,4 +89,20 @@ public boolean validatePredicate(@Name("predicate") boolean predicate, @Name("me
 
         return true;
     }
+
+    @UserFunction
+    @Description("apoc.util.decompress(compressed, {config}) | return a string from a compressed byte[] in various format")
+    public String decompress(@Name("data") List<Long> data, @Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws Exception {
+
+        CompressionConfig conf = new CompressionConfig(config);
+        return conf.getCompressionAlgo().decompress(data, conf.getCharset());
+    }
+
+    @UserFunction
+    @Description("apoc.util.compress(string, {config}) | return a compressed byte[] in various format from a string")
+    public List<Long> compress(@Name("data") String data, @Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws Exception {
+
+        CompressionConfig conf = new CompressionConfig(config);
+        return conf.getCompressionAlgo().compress(data, conf.getCharset());
```

Comment:
questo sulla base dei cambiamenti che ti ho chiesto diventa:
```suggestion
        return CompressionAlgo.valueOf(conf.getCompressionAlgo(), conf.getCharset())
```